docs/guides/example_messaging.md

```diff
@@ -0,0 +1,331 @@
+[**UDMI**](../../) / [**Docs**](../) / [**Guides**](./) / [Example](#)
+
+# UDMI example
+
+## Contents
+
+- [Introduction](#introudction)
+- [Physical Setup](#phyiscal-setup)
+- [Message Flow](#message-flow)
+
+## Introduction
+
+**Recommended reading**
+
+- [Tech primer](../tech_primer.md)
+- [UDMI documentation ](../)
+- [Compliance](../specs/compliance.md)
+
+This document describes a real world basic implementation of UDMI using a fan coil unit
+system as an example. Only basic messaging functionality is demonstrated and not complete functionality. 
+
+## Physical Setup
+
+For this example, the sample fan coil unit system comprises:
+- Fan
+- Cooling coil and valve
+- Supply & return air temperature sensorS
+- Fan coil unit controller
+
+![example-systems-diagram](images/example-systems-diagram.png)
+
+UDMI is implemented at the fan coil unit controller, which has an internet connection, and an MQTT
+client which supports UDMI.
+
+The system has the following physical points, represented as analogue inputs/outputs on the controller
+- `cooling_valve_percentage_command`
+- `supply_air_temperature_sensor`
+- `return_air_temperature_sensor`
+- `fan_speed_percentage_command`
+
+The system additionally comprises the following virtual points
+- `zone_air_temperature_setpoint`
+
+These points have been named according to [Digital Buildings Ontology (DBO)](../tech_primer.md) at
```

Comment:
So does UDMI not have any relationship to DBO?

Reply:
There is a loose coupling between UDMI and DBO, and it's not completely specified yet. The real statement would be something like "these points would be named according to the specified DBO entity type," but that's not quite instantiated yet (no way to specify the entity type).  The key difference is the handling of unspecified types -- since in that case UDMI would not care (which is different than a strong statement that the points have been named accordingly). So, again, for the perspective of an HVAC implementation then yes, this would be the case, but from UDMIs perspective it's "only when indicated"